apps/api/src/modules/user/user.repository.ts

```diff
@@ -17,4 +18,70 @@ export class UserRepository {
       where: { castingId },
     })
   }
+
+  async getUsers(params: {
+    skip?: number
+    take?: number
+    cursor?: Prisma.UserWhereUniqueInput
+    where?: Prisma.UserWhereInput
+    orderBy?: Prisma.UserOrderByWithRelationInput
+  }): Promise<User[]> {
+    return await this.prisma.user.findMany({
+      ...params,
+    })
+  }
+
+  async getPendingUsers(params: {
+    skip?: number
+    take?: number
+    cursor?: Prisma.UserWhereUniqueInput
+    where?: Prisma.UserWhereInput
+    orderBy?: Prisma.UserOrderByWithRelationInput
+  }): Promise<PendingUserDto[]> {
```

Comment:
are we gonna filter pendingUser If not these parameter should be remove

Reply:
according to the design, we don't need to filter, so I will remove these parameters.